### ISSUE TITLE
train saehd.bat stuck after pressing key to continue

### ISSUE THREAD

**Issue by pjflippy123** [iperov/DeepFaceLab#5473]
Running trainer.

[new] No saved models found. Enter a name of a new model :
new

Model first run.

Choose one or several GPU idxs (separated by comma).

[CPU] : CPU
  [0] : NVIDIA GeForce RTX 2070

[0] Which GPU indexes to choose? :
0

[1] Autobackup every N hour ( 0..24 ?:help ) :
1
[n] Write preview history ( y/n ?:help ) :
n
[0] Target iteration :
0
[y] Flip SRC faces randomly ( y/n ?:help ) :
y
[y] Flip DST faces randomly ( y/n ?:help ) :
y
[4] Batch_size ( ?:help ) :
4
[96] Resolution ( 64-640 ?:help ) :
96
[wf] Face type ( h/mf/f/wf/head ?:help ) :
wf
[df] AE architecture ( ?:help ) :
df
[256] AutoEncoder dimensions ( 32-1024 ?:help ) :
256
[64] Encoder dimensions ( 16-256 ?:help ) :
64
[64] Decoder dimensions ( 16-256 ?:help ) :
64
[22] Decoder mask dimensions ( 16-256 ?:help ) :
22
[y] Masked training ( y/n ?:help ) :
y
[n] Eyes and mouth priority ( y/n ?:help ) :
n
[n] Uniform yaw distribution of samples ( y/n ?:help ) :
n
[n] Blur out mask ( y/n ?:help ) :
n
[y] Place models and optimizer on GPU ( y/n ?:help ) :
y
[n] Use AdaBelief optimizer? ( y/n ?:help ) :
n
[n] Use learning rate dropout ( n/y/cpu ?:help ) :
n
[y] Enable random warp of samples ( y/n ?:help ) :
y
[0.0] Random hue/saturation/light intensity ( 0.0 .. 0.3 ?:help ) :
0.0
[0.0] GAN power ( 0.0 .. 5.0 ?:help ) :
0.0
[0.0] 'True face' power. ( 0.0000 .. 1.0 ?:help ) :
0.0
[0.0] Face style power ( 0.0..100.0 ?:help ) :
0.0
[0.0] Background style power ( 0.0..100.0 ?:help ) :
0.0
[none] Color transfer for src faceset ( none/rct/lct/mkl/idt/sot ?:help ) :
none
[n] Enable gradient clipping ( y/n ?:help ) :
n
[n] Enable pretraining mode ( y/n ?:help ) :
n
Initializing models: 100%|###############################################################| 5/5 [00:01<00:00,  2.71it/s]
Loading samples: 100%|############################################################| 2256/2256 [00:04<00:00, 503.05it/s]
Loading samples: 100%|##############################################################| 816/816 [00:01<00:00, 461.83it/s]
Press any key to continue . . .

gets stuck here 

![image](https://user-images.githubusercontent.com/99305748/153113928-411bb5b9-a9a7-4e5f-9dde-524ac9b9e8e9.png)

uses the same amount of ram for 8 hours i had it open but never once opened the trainer or got past this stage

using i7 with a rtx 2070 16 gb of ram

**Comment by mapeles** [comment]
I am having the same problem
win11
geforce driver 496.49

**Comment by Julia222222** [comment]
I think, but I am not sure, you have to train Quick96 first before do a SHAED training, because the first time I used deepfacelab I did so and  it worked, and the second time I used it I didn´t train Quick96 at first and it didn´t work, now I am in the process of training Quick96 in thisone to see if I can train SHAED after

**Comment by syedhaider1961** [comment]
I dont have money so sorry I may quit

**Comment by Julia222222** [comment]
I find the solution, you have to use the CPU in the first few interactions and then save and change to GPU

**Comment by mapeles** [comment]
> 
That method works for quick96 but not for shahd

**Comment by mapeles** [comment]
> I think, but I am not sure, you have to train Quick96 first before do a SHAED training, because the first time I used deepfacelab I did so and it worked, and the second time I used it I didn´t train Quick96 at first and it didn´t work, now I am in the process of training Quick96 in thisone to see if I can train SHAED after

maybe this could be the solution

**Comment by paulcanning** [comment]
> I find the solution, you have to use the CPU in the first few interactions and then save and change to GPU

Using the CPU did start the training, but after saving and then trying to retrain using the GPU, after displaying the current training model status, I got `Press any key to continue` again and nothing happens.

Using the latest version for windows.

**Comment by Alos109** [comment]
Hello everyone 
I have also noticed that Saehd does not work with the latest Nvidia drivers. What happens is training Wont open. So I tried gong back to drivers August 2021 and it Works. Do you Guys think They wil fix it ?

**Comment by luckypig3400** [comment]
> Hello everyone I have also noticed that Saehd does not work with the latest Nvidia drivers. What happens is training Wont open. So I tried gong back to drivers August 2021 and it Works. Do you Guys think They wil fix it ?

Thanks a lot 😀😀😀
## This works for me properly ~!

**Comment by mrlunk** [comment]
I definately hope this will be fixed.

**Comment by EwenBernard** [comment]
> > Hello everyone I have also noticed that Saehd does not work with the latest Nvidia drivers. What happens is training Wont open. So I tried **gong back to drivers August 2021** and it Works. Do you Guys think They wil fix it ?
> 
> Thanks a lot 😀😀😀
> 
> ## This works for me properly ~!
> The driver version I downgrade to is **471.68** which was released at **20210810**.

Works with 471.68

**Comment by MadeByKit** [comment]
> > Hello everyone I have also noticed that Saehd does not work with the latest Nvidia drivers. What happens is training Wont open. So I tried **gong back to drivers August 2021** and it Works. Do you Guys think They wil fix it ?
> 
> Thanks a lot 😀😀😀
> ## This works for me properly ~!
> 
> The driver version I downgrade to is **471.68** which was released at **20210810**.

does this work on a low pc like gtx 1650?

**Comment by ghost** [comment]
Issue solved / already answered (or it seems like user error), please close it.
